submissions/morenko/memory-game/js/index.js

```diff
@@ -0,0 +1,182 @@
+// создадим массив с картами
+const listCards = [
+    {
+        name: 1,
+        front: 'assets/front/1.jpg'
+    },
+    {
+        name: 2,
+        front: 'assets/front/2.jpg'
+    },
+    {
+        name: 3,
+        front: 'assets/front/3.jpg'
+    },
+    {
+        name: 4,
+        front: 'assets/front/4.jpg'
+    },
+    {
+        name: 5,
+        front: 'assets/front/5.jpg'
+    },
+    {
+        name: 6,
+        front: 'assets/front/6.jpg'
+    },
+    {
+        name: 7,
+        front: 'assets/front/7.jpg'
+    },
+    {
+        name: 8,
+        front: 'assets/front/8.jpg'
+    },
+];
+//console.log(cards);
```

Comment:
You should remove such code from PR before making it. Some companies allow it, but it is considered as a bad practice. Your code looks dirty :)